src/main/java/io/kestra/plugin/amqp/AbstractAmqpConnection.java

```diff
@@ -24,7 +25,9 @@ public abstract class AbstractAmqpConnection extends Task implements AmqpConnect
     private String port;
     private String username;
     private String password;
-    private String virtualHost;
+
+    @Builder.Default
+    private String virtualHost = "/";
```

Comment:
Seems to be a breaking change and not needed, WDYT?

Reply:
Yes, as expected, the virtual host cannot be null:
```
java.lang.IllegalStateException: Invalid configuration: 'virtualHost' must be non-null.
```
But anyway, I've changed my approach, and let the AMQP client manage its default values, so I removed this one.